src/pg/sql/44_observatory_geometries.sql

```diff
@@ -0,0 +1,163 @@
+-- Returns the polygon(s) that overlap with the input geometry.
+-- Input:
+-- :param geom geometry: input geometry
+-- :param boundary_id text: table to get polygon from (can be approximate name)
+-- :param use_literal boolean: use the literal table name (defaults to true)
+
+-- From an input point geometry, find the boundary which intersects with the centroid of the input geometry
+
+CREATE OR REPLACE FUNCTION Andy_OBS_GetGeometry(
+  geom geometry(Geometry, 4326),
+  boundary_id text DEFAULT '"us.census.tiger".census_tract', -- TODO: from a specified column id list (e.g., list of available catalog, see OBS_List)
+  time_span text DEFAULT '2009 - 2013')
```

Comment:
See above.  This should default to `NULL` and then we can have logic `TODO` for choosing the "latest".